endpoint/domain_filter.go

```diff
@@ -226,3 +228,10 @@ func (df DomainFilter) MatchParent(domain string) bool {
 	}
 	return false
 }
+
+// normalizeDomain converts a domain to a canonical form, so that we can filter on it
+// it: trim "." suffix, get Unicode version of domain complient with Section 5 of RFC 5891
+func normalizeDomain(domain string) string {
+	s, _ := idna.Lookup.ToUnicode(strings.TrimSuffix(domain, "."))
```

Comment:
Wdyt about catching the error and, at least, display a warning ?

Reply:
The function is opinionated to return a partial string on errors ([source](https://cs.opensource.google/go/x/net/+/refs/tags/v0.28.0:idna/idna10.0.0.go;l=30-36)) so I judged that we could ignore it if just to avoid log spam should something happen. But you we could log it as a `warning` or `debug`  depending what you prefer?